Application/Education/Certificate/Generator/Service/DataCertificate/SDataGym.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+
+namespace SPHERE\Application\Education\Certificate\Generator\Service\DataCertificate;
+
+
+use SPHERE\Application\Education\Certificate\Generator\Service\Data;
+use SPHERE\Application\Education\Lesson\Division\Division;
+
+class SDataGym
```

Comment:
Codacy found an issue: [The class SDataGym has an overall complexity of 54 which is very high. The configured complexity threshold is 50.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)